app/javascript/admin/controllers/article_controller.js

```diff
@@ -1,6 +1,5 @@
 import { Controller } from 'stimulus';
 
-// eslint-disable-next-line no-restricted-syntax
```

Comment:
Removing the eslint exceptions removes the burden of devs having to put it anytime they create a Stimulus controller.